alpine/elasticsearch/2.x/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM nowait/java:openjre8-alpine
+MAINTAINER Nowait <devops@nowait.com>
+
+ENV ELASTICSEARCH_VERSION 2.3.2
```

Comment:
Just curious why there are 3 separate ENV lines instead of making it just one.  Could it be combined with the ENV statement below as well?